internal/host/installcmd.go

```diff
@@ -227,3 +238,36 @@ func (i *installCmd) getDiskUnbootableCmd(ctx context.Context, host models.Host)
 	}
 	return formatCmds, nil
 }
+
+func constructHostInstallerArgs(cluster *common.Cluster, host *models.Host) (string, error) {
+	if cluster.ImageInfo.StaticIpsConfig == "" {
+		return host.InstallerArgs, nil
```

Comment:
Does it mean that the rest of the function isn't needed when no static IPs are used?

Reply:
yes.